### PR TITLE
sample-apiserver: add v1beta1 with advanced conversion example from v1alpha1

### PR DESCRIPTION
**Pull request by sttts**
Follow-up of discussion in https://github.com/kubernetes/kubernetes/pull/60594.

### CODE REVIEW

File: staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder/strategy.go [hunk 1]
```diff
@@ -77,7 +78,8 @@ func (flunderStrategy) PrepareForUpdate(ctx genericapirequest.Context, obj, old
 }
 
 func (flunderStrategy) Validate(ctx genericapirequest.Context, obj runtime.Object) field.ErrorList {
-	return field.ErrorList{}
+	flunder := obj.(*wardle.Flunder)
+	return validation.ValidateFlunder(flunder, )
```
Comment (liggitt) [inline staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder/strategy.go]: compile error?

**Comment by ixdy** [comment]
I think this may have broken the cross build. cc @rmmh

**Comment by liggitt** [comment]
weird... looks like update-bazel doesn't mess with files under k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1

**Comment by liggitt** [comment]
Apparently we have multiple ways of building :-/

**Comment by liggitt** [comment]
Fix is in https://github.com/kubernetes/kubernetes/issues/62748

**Pull request merged**